fe/fe-core/src/main/java/com/starrocks/sql/analyzer/PrepareAnalyzer.java

```diff
@@ -47,6 +54,12 @@ public void analyze(PrepareStmt prepareStmt) {
     }
 
     public void analyze(ExecuteStmt executeStmt) {
+        // prepare stmt key
+        if (!session.getSessionVariable().isEnablePrepareStmt() && executeStmt != null) {
+            throw new StarRocksPlannerException(ErrorType.INTERNAL_ERROR, "execute statement can't be executed, maybe "
+                    + "need set enable_prepare_stmt=true");
+        }
+
```

Comment:
could we reject prepare/execute stmt earlier when it's parsed? in connectprocessor.java